torchmultimodal/modules/encoders/albef_text_encoder.py

```diff
@@ -233,6 +206,10 @@ def __init__(
         self.key = nn.Linear(hidden_size, self.all_head_size)
         self.value = nn.Linear(hidden_size, self.all_head_size)
 
+        self.self_attention = SelfAttention()
```

Comment:
Random question not related to this PR: seems that SelfAttention init has shape as a mandatory arg, when in reality this is only used in the causal case. I guess it's not throwing any errors that we're not passing it (not sure why?), but maybe we should change to make it optional and check in the causal case?

Reply:
I think this was already updated in a separate PR or a currently open one. Either way you're right this will be addressed.